jupyter_notebooks/Book_1_Intro_Williamson_2.ipynb

```diff
@@ -0,0 +1,380 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ec4b5fe",
+   "metadata": {},
+   "source": [
+    "# Notebook 1: Introduction with Shallow Water - Williamson 2 Test Case"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5371527b",
+   "metadata": {},
+   "source": [
+    "This notebook provides an introduction to Gusto. We will go through the example of the Williamson 2 test case to demonstrate how to set up the problem and run it. This comes from the paper: \\\n",
+    "David L. Williamson, John B. Drake, James J. Hack, Rüdiger Jakob, Paul N. Swarztrauber,\n",
+    "A standard test set for numerical approximations to the shallow water equations in spherical geometry,\n",
+    "Journal of Computational Physics,\n",
+    "Volume 102, Issue 1,\n",
+    "1992,\n",
+    "Pages 211-224,\n",
+    "ISSN 0021-9991,\n",
+    "https://doi.org/10.1016/S0021-9991(05)80016-6."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "664c0cba",
+   "metadata": {},
+   "source": [
+    "The Williamson 2 test case solves the shallow water equations. This corresponds to the following set of momentum and continuity equations, for a velocity $\\textbf{u}$ and free surface, $\\eta$:\n",
+    "$$\\textbf{u}_t + f \\textbf{u}^{\\perp} + g \\nabla \\eta + (\\textbf{u} \\cdot \\nabla) \\textbf{u} = 0$$\n",
+    "$$\\eta_t + H(\\nabla \\cdot \\textbf{u}) + \\nabla [\\textbf{u} (\\eta - b)] = 0$$\n",
+    "\n",
+    "The parameters we need to specify are the Coriolis force, $f$, gravitational constant, $g$, and mean depth, $H$.\n",
+    "$\\newline$\n",
+    "We can neglect boundary conditions, due to solving on a spherical domain. \\ \n",
+    "Williamson 2 is called 'Global Steady State Nonlinear Zonal Geostrophic Flow', so the solution should not change over time."
```

Comment:
Maybe we could rewrite this as "The second Williamson test case models zonal geostrophic flow. This is a steady-state solution of the shallow-water equations, so the true solution does not change with time."